plugins/woocommerce-admin/client/core-profiler/index.tsx

```diff
@@ -1147,15 +1151,69 @@ export const coreProfilerStateMachineDefinition = createMachine( {
 								completed: true,
 							} );
 						},
-						onDone: {
-							actions: 'redirectToWooHome',
-						},
+						onDone: [
+							{
+								target: 'isJetpackConnected',
+								cond: ( _context ) => {
```

Comment:
small nit: `_` is used to denote unused parameters so this should be `context`

Also, could we move these inline implementations out so that it is easier to read the states?

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/38674/commits/bced7e54accba2bda036ed6ce73f4be02b747991